### DISCUSSION TITLE
My images doesn't appear.

### DISCUSSION THREAD

**Discussion by ghost** [community/community#65579] [New to GitHub] [answered]
### Select Topic Area

Show & Tell

### Body

Hello guys, I'm new to Github, I have a website which I created and its working fine, the images are stored on a folder which I used for the website.

So, I've hosted it on Github but I doesn't load the images..I'm confused and don't know how to navigate the environment to get it done.

**Comment by radar-blip** [comment]
There isn't a folder, or any images in your repository. Did you forget to upload them?

**Reply by ghost** [reply]
I didn't upload the images to the repository..
How can I add it now or will I have to add it back when I did: git clone https://............

Inside the folder that I moved my index and style.css?

**Reply by radar-blip** [reply, marked as answer]
Yes upload the images to the same place as the other files.

**Reply by ghost** [reply]
OK..I'll try it out
Thanks for your time😊

**Reply by radar-blip** [reply]
You're welcome. Good luck!

**Reply by ghost** [reply]
Thanks a lot, it worked😊

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Joses47 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!
